### ISSUE TITLE
"CONTRIBUTING.md" and  "CONDUCT.md" should be skipped by check.py 

### ISSUE THREAD

**Issue by iglpdc** [swcarpentry/DEPRECATED-lesson-template#209]
When running `tools/check.py` in the git-novice lesson, I get:

```
ERROR:root:Validation failed for /Users/ivan/Documents/swc/git-novice/CONDUCT.md: Could not automatically identify correct template.
ERROR:root:Validation failed for /Users/ivan/Documents/swc/git-novice/CONTRIBUTING.md: Could not automatically identify correct template.
```

I think that those two files should be added to "SKIP_FILES" in `tools/check.py`.

I could do a PR later if someone confirms this.

**Comment by abought** [comment]
They're not skipped? Let me take a look later; this has been sitting in my Mail Folder of Shame for too long now.

**Comment by abought** [comment]
Ah... looks like this functionality was added in https://github.com/swcarpentry/lesson-template/commit/c7c6aaa6c193cd8250127daff5f19024096f6cf5 , but then accidentally reverted during the last reconciliation of `core` and `gh-pages` four days ago.

See: https://github.com/swcarpentry/lesson-template/commit/e22aa93637d9ca8aad4657cf1252a541c7a4dea1

Since the changes are technically already part of the commit history for both branches, my original branch can't be used to create a new pull request. I'll generate a new branch in a moment and re-apply the changes.
https://github.com/abought/swc-lesson-template/tree/ignore_contrib

**Comment by abought** [comment]
Should be fixed once #210 is merged. Thanks for the sharp eyes and quick feedback. (caught in just a few days!)

**Comment by iglpdc** [comment]
Thanks @abought! That was quick.
